src/50_Visual_Effects/Basics_of_orientation_effects.html

```diff
@@ -0,0 +1,304 @@
+
+  In this tutorial, we will go through some of these use-cases in detail.
+-->
+
+<!--
+  ## Components
+  [amp-orientation-observer](https://www.ampproject.org/docs/reference/components/amp-orientation-observer) is a
+  functional component that monitors the orientation of the device as it moves,
+  and dispatches `alpha`, `beta` and `gamma` events dependant on what axis the device has moved in. 
+  For more details on DeviceOrientationEvents read: https://developer.mozilla.org/en-US/docs/Web/API/Detecting_device_orientation
+
+  These events in return can be used to `seek` animation timelines
+  defined by [amp-animation](https://www.ampproject.org/docs/reference/components/amp-animation)
+  to create orientation based effects.
+
+  [amp-animation](https://www.ampproject.org/docs/reference/components/amp-animation) is a UI component
+  that relies on [Web Animations API](https://www.w3.org/TR/web-animations/) to define and
+  run keyframe animations in AMP documents.
+-->
+
+<!-- -->
+<!doctype html>
+<html ⚡>
+
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- ## Setup -->
+  <!--  Import amp-position-observer extension -->
+  <script async custom-element="amp-orientation-observer" src="https://cdn.ampproject.org/v0/amp-orientation-observer-0.1.js"></script>
+
+  <!--  Import amp-animation extension -->
+  <script async custom-element="amp-animation" src="https://cdn.ampproject.org/v0/amp-animation-0.1.js"></script>
+
+  <link rel="canonical" href="<%host%>/visual_effects/basics_of_orientation_effects/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+  <!--
+    ## Styles
+
+    The CSS used for these samples are included here for reference.
+
+    These rules are simply needed to make the samples work but are not fundamental
+    to the concepts covered here.
+  -->
+  <style amp-custom>
+    #scrollText {
+      font-family: Helvetica, sans-serif;
+      padding: 10px;
+      max-width: 412px;
+      height: 500px;
+      margin: auto;
+      overflow: hidden;
+    }
+
+    #ipsum {
+      background-image: linear-gradient(white, deeppink);
+    }
+
+    amp-img img {
```

Comment:
Can you make this rule more specific, otherwise it'll mess with the generated page.